frontend/packages/operator-lifecycle-manager/src/style.scss

```diff
@@ -54,6 +58,20 @@ $co-affinity-row-margin: 15px;
   display: flex;
 }
 
+.pf-c-table .pf-m-wrap {
+  white-space: normal;
+  vertical-align: top;
+}
+
+.pf-c-table__sort.pf-m-wrap .pf-c-button {
+  display: flex;
+  white-space: normal;
+}
+
+.pf-c-table__sort-indicator {
+  line-height: normal !important;
+}
+
```

Comment:
This will impact *all* tables. We don't want that. Use BEM classes to scope these changes to the single table we need.

Reply:
These are PF 4 styles we don't seem to be pulling in (PatternFly officially supports wrappable, sortable columns, but we don't seem to have that support within OpenShift out of the box for some reason). I can scope these, but do you know if there's a reason we don't already have support for this? It would be ideal if I can just drop these.

Reply:
> These are PF4 styles we don't seem to be pulling in (PatternFly officially supports wrappable, sortable columns, but we don't seem to have CSS support within OpenShift out of the box for some reason). I can scope these, but do you know if there's a reason we don't already have support for this? It would be ideal if I could just drop these.

Are we sure we have the right Patternfly version for that? It was added recently.

cc @sg00dwin @rhamilto

Reply:
It looks like it was added back in August and we're past those versions: https://www.patternfly.org/v4/documentation/react/overview/release-notes#201906-release-notes-2019-08-13

Reply:
> It looks like it was added back in August and we're past those versions: https://www.patternfly.org/v4/documentation/react/overview/release-notes#201906-release-notes-2019-08-13

We need this fix, which is more recent: https://github.com/patternfly/patternfly/pull/2668

Reply:
It looks like that was part of release 2.65.2 and we're on 2.65.3?

https://www.patternfly.org/v4/documentation/core/overview/release-notes#202002-release-notes-2020-02-18

Reply:
Seem like we need to do one of the following:
1. bump PF to pick up the fix.
2. scope it to just this instance for now.
3. if not scoped, move it out of `packages/operator-lifecycle-manager`, if it's going to affect all tables until we can bump PF.

Reply:
@rebeccaalpert can you remove these styles from this PR and we'll handle it in #5078 ?

Reply:
All set!